doc/api/quic.md

```diff
@@ -260,12 +260,9 @@ added: REPLACEME
       IPv6 address or a host name. If a host name is given, it will be resolved
       to an IP address.
     * `port` {number} The local port to bind to.
-    * `type` {string} Either `'udp4'` or `'upd6'` to use either IPv4 or IPv6,
-      respectively. **Default**: `'udp4'`.
-    * `ipv6Only` {boolean} If `type` is `'udp6'`, then setting `ipv6Only` to
-      `true` will disable dual-stack support on the UDP binding -- that is,
-      binding to address `'::'` will not make `'0.0.0.0'` be bound. The option
-      is ignored if `type` is `'udp4'`. **Default**: `false`.
+    * `type` {string} Can be one of `'udp4'`, `'upd6'`, or `'udp6-only'` to
+      use IPv4, IPv6, or IPv6 with dual-stack mode disabled.
+      **Default**: `'udp4'`.
```

Comment:
If we do this, should we update `net` and `dgram` to also accept `'udp6-only'`? It seems odd that we deviate from the `type + ipv6Only` option pairing only in this place

Reply:
Yes, eventually we should, yes. I just don't want to do it in this PR.